Tests/SwiftCliCoreTests/ChartifyTest.swift

```diff
@@ -14,13 +14,13 @@ class ChartifyFinishedTest: XCTestCase {
     func testRunCallsMakeChartIfInputIsValid() throws {
         let mock = MockChartConstructor()
 
-        Chartify(inputValidator: InputValidator(), chartConstructor: mock, fileValidator: FileValidator()).run(userInput: ["k1"])
+        Chartify(inputValidator: InputValidator(patternNormalizer: PatternNormalizer(), nestedArrayBuilder: NestedArrayBuilder()), chartConstructor: mock, fileValidator: FileValidator()).run(userInput: ["k1"])
 
         expect(mock.wasMakeChartCalled).to(equal(true))
     }
     func testRunCallsFileValidatorIfFileIsIncluded() throws {
         let mock = MockFileValidator()
-        Chartify(inputValidator: InputValidator(), chartConstructor: ChartConstructor(), fileValidator: mock).run(userInput: ["k1"], file: Optional("example.txt"))
+        Chartify(inputValidator: InputValidator(patternNormalizer: PatternNormalizer(), nestedArrayBuilder: NestedArrayBuilder()), chartConstructor: ChartConstructor(), fileValidator: mock).run(userInput: ["k1"], file: Optional("example.txt"))
```

Comment:
Nonblocking, but you could probably just initialize a `Chartify` with all three mocks in a `setUp` here.